tests/contrib/langchain/test_langchain.py

```diff
@@ -979,7 +775,7 @@ def test_chain_logs(langchain, ddtrace_config_langchain, request_vcr, mock_logs,
                     "service": "",
                     "status": "info",
                     "ddtags": "env:,version:,langchain.request.provider:openai,langchain.request.model:text-davinci-003,langchain.request.type:llm,langchain.request.api_key:...key>",  # noqa: E501
-                    "dd.trace_id": str(llm_span.trace_id),
+                    "dd.trace_id": hex(llm_span.trace_id)[2:],
```

Comment:
nit: This will ensure you will always get zero padded trace_ids (and without the 0x)
```suggestion
                    "dd.trace_id": f"{llm_span.trace_id:0{16}x}",
```

Reply:
So this is the same as `hex(trace_id)[2:]`? Will the zero padding ever produce a different result?